plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -298,13 +297,15 @@ def _get_and_save_file_units(filename, processing_function, tag, conduit, repo_i
     :param conduit:  provides access to relevant Pulp functionality
     :type  conduit:  pulp.plugins.conduits.upload.UploadConduit
 
-    :param repo_id:  id of the repo into which unit will be uploaded
-    :type  repo_id:  str
+    :param repo: The repository to import the package into
+    :type  repo: pulp.server.db.model.Repository
     """
+    repo_id = repo.repo_id
```

Comment:
Oh you are right, I misread this line to be `repo = repo.repo_id` instead. There is no problem here.